package.json

```diff
@@ -14,7 +14,8 @@
   },
   "scripts": {
     "test": "xo && mocha",
-    "bench": "matcha bench.js"
+    "bench": "matcha bench.js",
+    "jshint": "jshint index.js"
```

Comment:
Check the CI result. https://travis-ci.org/sindresorhus/object-assign/jobs/108365657#L151

`xo` is of course not installed globally on Travis CI environment, but `npm test` runs without `Command not found` error.